public/app/features/migrate-to-cloud/api/endpoints.gen.ts

```diff
@@ -5,7 +5,7 @@ const injectedRtkApi = api.injectEndpoints({
       query: () => ({ url: `/cloudmigration/migration` }),
     }),
     createMigration: build.mutation<CreateMigrationApiResponse, CreateMigrationApiArg>({
-      query: (queryArg) => ({ url: `/cloudmigration/migration`, method: 'POST', body: queryArg.cloudMigrationRequest }),
+      query: () => ({ url: `/cloudmigration/migration`, method: 'POST' }),
```

Comment:
uhhhh

Reply:
Not sure where the swagger annotations for this struct went, very weird. You'll have to add one for `CloudMigrationRequest` in `api.go` to fix drone.

Reply:
Done